doc/Set-custom-workspace.md

```diff
@@ -42,7 +42,9 @@ Another better solution if you faced this problem is to use a custom workspace d
 
 You could set the workspace directory when cloning the tfs repository using the `--workspace` option :
 
-    git tfs clone http://server/tfs $/Project/trunk project --workspace="c:\ws"
+```DOS
```

Comment:
AFAIK there is no `DOS` in the supported list of languages.
https://github.com/github-linguist/linguist/blob/master/lib/linguist/languages.yml

Maybe change this to `sh` ?

Reply:
Agreed..

How about

```Shell
```

https://github.com/github-linguist/linguist/blob/master/lib/linguist/languages.yml#L6472

?